txbits/app/models/UserModel.scala

```diff
@@ -57,7 +57,7 @@ class UserModel(val db: String = "default") {
   }
 
   def addFakeMoney(uid: Long, currency: String, amount: BigDecimal) = DB.withConnection(db) { implicit c =>
-    if (Play.current.configuration.getBoolean("fakeexchange").get) {
+    if (globals.fakeExchange) {
```

Comment:
Right. The globals object is set up once when the tests start. Within a test you get a different play server started up for each test with different configs. Look at the helper classes in the tests directory.